### DISCUSSION TITLE
Titlebars persist even after trying to disable

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5738] [Q&A]
I still have the title bars like the one shown below even though I have disabled it. 
![image](https://github.com/i3/i3/assets/113855995/512e5c58-d9b5-404e-afa0-11cd4eb82cc9)

[config.txt](https://github.com/i3/i3/files/13196766/config.txt)

**Comment by okraits** [comment]
Two things to note:

- The config directive `default_border`doesn't affect existing windows, only new ones.
- Windows in containers with the tabbed or stacked layout will still have titlebars.
